src/papyrus-lang-vscode/src/ExtensionConfigProvider.ts

```diff
@@ -8,7 +8,9 @@ export interface IGameConfig {
     readonly creationKitIniFiles: string[];
     readonly installPath: string;
     readonly ignoreDebuggerVersion: boolean;
-    readonly modDirectoryPath: string;
+    readonly modDirectoryPath: string;              // For supporting mod managers managing the debug plugin
```

Comment:
I'm thinking that this might not really be the right name for this. We're really just talking about a custom base path for XSE plugins regardless of the original use case.

Reply:
I don't know of any other use case for this. Since the game only supports one data directory the only reason for referring to an alternate one is if it's some sort of overlay, and I only know of mod managers supplying those.